new_platforms/vscode-extension/README.md

```diff
@@ -103,17 +95,26 @@ the host and enclave symbols into an instance of the debugger.
 
 * Install [Visual Studio Code](https://code.visualstudio.com/)
 * Install [git](https://git-scm.com/downloads)
-    * On Windows, please make sure that long paths are enabled: `git config --system core.longpaths true`
-* Install [CMake 3.12 or higher](https://cmake.org/download/)
-* On Linux, install the required build components
-
-      sudo apt update && sudo apt install -y build-essential cmake gcc-arm-linux-gnueabihf gcc-aarch64-linux-gnu g++-arm-linux-gnueabihf g++-aarch64-linux-gnu gdb-multiarch python
-
-* Make sure that the [Native Debug extension](https://marketplace.visualstudio.com/items?itemName=webfreak.debug) is installed.
+    * On Windows, please make sure that long paths are enabled: `git config --global core.longpaths true`
+* On Linux
+
+    * Make sure that the [Native Debug extension](https://marketplace.visualstudio.com/items?itemName=webfreak.debug) is installed.
+    * Install [CMake 3.12 or higher](https://cmake.org/download/)
+    * Install the required build components: 
+        
+          sudo apt update && sudo apt install -y build-essential cmake gcc-arm-linux-gnueabihf gcc-aarch64-linux-gnu g++-arm-linux-gnueabihf g++-aarch64-linux-gnu gdb-multiarch python
+    
 * Ensure that the requirements are met for the [Azure IoT Edge extension](https://marketplace.visualstudio.com/items?itemName=vsciot-vscode.azure-iot-edge):
-    * Ensure that [Docker is installed and running: https://docs.docker.com/install/](https://docs.docker.com/install/).
-        * On Linux, if you are seeing permissions issues when connecting to the docker daemon, this has helped some people: `sudo usermod -a -G docker $USER`
-    * Ensure that the [iotedgehubdev](https://pypi.org/project/iotedgehubdev/) tool is installed
+    * [Docker is installed and running: https://docs.docker.com/install/](https://docs.docker.com/install/).
+        * On Linux, to cross-build a container (i.e. if your host is amd64 and your container is arm32v7), you will need to enable cross-building.  Adding these packages solved the cross-build issue on Ubuntu 19.04:
+            
+              sudo apt-get install -y qemu qemu-user-static qemu-user binfmt-support
+        
+        * On Linux, if you are seeing permissions issues when connecting to the docker daemon, this has helped some people:
```

Comment:
Consider rephrasing as:

On Linux, if you are seeing permission issues related to connecting to the Docker daemon, add your Linux user to the docker group. This will allow your user to connect and issue commands to the Docker daemon:

Reply:
ok